src/Loader.php

```diff
@@ -1081,7 +1081,10 @@ public static function add_component_settings( $settings ) {
 		if ( ! empty( $preload_settings ) ) {
 			$setting_options = new \WC_REST_Setting_Options_V2_Controller();
 			foreach ( $preload_settings as $group ) {
-				$group_settings   = $setting_options->get_group_settings( $group );
+				$group_settings = $setting_options->get_group_settings( $group );
+				if ( $group_settings instanceof \WP_Error ) {
```

Comment:
We typically use [`is_wp_error()`](https://developer.wordpress.org/reference/functions/is_wp_error/) for this. (It's the same implementation) 😄

Reply:
Also curious what may have caused an entire settings group to not exist, but may also be worth adding in some error logging here if that's the case.

Reply:
@jeffstieler @joshuatf Thank you for the suggestions. It's also possible that one of the bug reporter's plugins is returning an invalid value to the filter.  

I also agree that it's worth adding some error logging logic here since we might be suppressing an error silently, but I'm not sure what approach would be best at the moment.

Since this is a low priority ticket, I will contact the bug reporter and ask for more details to figure out what's causing the issue. I will put this PR to [WIP].

Reply:
I've updated `is_wp_error()` change for now.

This is my first time seeing this issue, but this PR does not show my latest commits for some reason.

It does not list this commit https://github.com/woocommerce/woocommerce-admin/commit/3f4a4e5cba4e4709e273feb88c7e826ad5e0e13e

Reply:
> It does not list this commit 3f4a4e5

That's odd. Does it fix if you cherry-pick the  commit?

Reply:
> I also agree that it's worth adding some error logging logic here since we might be suppressing an error silently, but I'm not sure what approach would be best at the moment.

How about throwing an Exception here?

```
throw new Exception( 'Group Settings is WP_Error' );
```

It would halt further processing, helping protect against any possibility of data loss from the issue. I believe the user would see a friendly WordPress error message, and developers could see the exception details in the error log.